priv/repo/migrations/20200115013245_add_privacy_to_hubs.exs

```diff
@@ -0,0 +1,11 @@
+defmodule Ret.Repo.Migrations.AddPrivacyToHubs do
+  use Ecto.Migration
+
+  def change do
+    Ret.Hub.Privacy.create_type()
+
+    alter table(:hubs) do
+      add(:privacy, :hub_privacy, null: false, default: "private")
```

Comment:
after this runs, as a sanity check make sure this lands in the `ret0` schema. I made some changes to the migration tools that should do this automatically now, but in the past we had to specify it in the migration.